DISCUSSIONS.md

```diff
@@ -4,11 +4,11 @@
 
 Notes about why we need realms:
 
-* Realms allow virtualization of the language itself.
+* ShadowRealms allow virtualization of the language itself.
 * Same origin is seen as legacy from some implementers, but same origin iframes is available anyways.
   * It is also available in node via `vm` module.
 * Same origin is an atomic part of the web reality.
-* Varios examples of why Realms are needed:
+* Varios examples of why ShadowRealms are needed:
```

Comment:
Varios -> Various

Reply:
(sarcasm): _I was not using Oxford English._